en/docs/advance-topics/guides/enable-workflow.md

```diff
@@ -0,0 +1,50 @@
+# Adding User and Application Creation Workflow
+Attaching a custom workflow to the user and application creation processes allows admin users to manage these actions more effectively by providing an option to approve or reject each creation request. This ensures that user and application creation is fully controlled and monitored by administrators.
+
+Healthcare accelerator provides the following capabilities. 
+
+- View the user details in a user creation request. 
+- Option to approve/reject user creation request. 
+- View the application details in a application creation request. 
+- Option to approve/reject application creation request. 
+
+## How to enable the User/Application creation workflow 
+1. Follow the [Installation Steps](../../install-and-setup/manual.md) to install the Healthcare Accelerator. 
```

Comment:
Healthcare APIM Accelerator

Reply:
Fixed in https://github.com/wso2/open-healthcare-docs/pull/45/commits/bc3c197716619abfbc8849f455e744b9f0f18df5